ush/parsing_namelists_FV3.sh

```diff
@@ -19,6 +19,11 @@ DIAG_TABLE_APPEND=${DIAG_TABLE_APPEND:-${PARMgfs}/ufs/fv3/diag_table_aod}
 DATA_TABLE=${DATA_TABLE:-${PARMgfs}/ufs/MOM6_data_table.IN}
 FIELD_TABLE=${FIELD_TABLE:-${PARMgfs}/ufs/fv3/field_table}
 
+# set cdmbgwd
+if (( gwd_opt == 2 )) && [[ ${do_gsl_drag_ls_bl} == ".true." ]]; then
+  cdmbgwd=${cdmbgwd_gsl}
```

Comment:
Is there a reason this is no longer in the config file?

Reply:
This if statement was moved to parsing_namelists_FV3.sh because the ensemble forecasts was not getting the correct value. This value is set in config.ufs, and over-set in config.fcst. Since config.ufs is sourced in config.efcs after the config.fcst source, the value was reset to the config.ufs value